includes/ziti/ziti.h

```diff
@@ -236,21 +239,27 @@ typedef struct ziti_options_s {
     tls_context *tls;
 
     const char **config_types;
+
+    /*! \deprecated */
     ziti_init_cb init_cb;
+
+    /*! \deprecated */
     ziti_service_cb service_cb;
 
     long refresh_interval; //the duration in seconds between checking for updates from the controller
     rate_type metrics_type; //an enum describing the metrics to collect
 
     int router_keepalive;
 
-    void *ctx;
-
     //posture query cbs
     ziti_pq_mac_cb pq_mac_cb;
     ziti_pq_os_cb pq_os_cb;
     ziti_pq_process_cb pq_process_cb;
     ziti_pq_domain_cb pq_domain_cb;
+
+    void *app_ctx;
```

Comment:
shouldn't all the other callbacks get this context too? or could we somehow bake it into the ziti_context?